core/message-tags-3.3.md

```diff
@@ -179,6 +201,19 @@ A message sent by a client with the `+example-client-tag` client-only tag:
 
 ---
 
+A message with the same tags sent with both the normal and JSON encodings for comparison:
+
+    @tag=value;+client-tag=value\swith\sspaces PRIVMSG #channel :Message
+    @{"tag":"value","+client-tag":"value\swith\spaces"} PRIVMSG #channel :Message
```

Comment:
I don't see `\s` as valid escape for space in JSON (https://www.ietf.org/rfc/rfc4627.txt)

Reply:
I just noticed that the example is `"value\swith\spaces"` should be correct when it's `"value\swith\sspaces"`